src/index.less

```diff
@@ -1,16 +1,19 @@
-@import url('https://fonts.googleapis.com/css2?family=ABeeZee&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

Comment:
Should we maybe just stick with one font throughout our app? I saw that Roboto is still used in theme.less

Reply:
Yeah I think we should -- as part of clean up in the next week or so I think we need to do an audit of styling on the frontend to make sure everything's consistent!

https://promopal.atlassian.net/browse/PP-100